src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -55,8 +55,8 @@ public HttpResponseMessage Invoke(string name, [FromBody] FunctionInvocation inv
         }
 
         [HttpGet]
-        [Route("admin/functions/{name}")]
-        public FunctionStatus Get(string name)
+        [Route("admin/functions/{name}/status")]
+        public FunctionStatus GetFunctionStatus(string name)
         {
             FunctionStatus status = new FunctionStatus();
             Collection<string> functionErrors = null;
```

Comment:
I know it's out of the scope of your changes, but on line 64, we'll probably always end up with a NullReferenceException in the case where we're unable to start since the "Instance" property is not set until after a successful start.